test/test_layers/test_layer_conv1x1.jl

```diff
@@ -2,8 +2,9 @@
 # Author: Philipp Witte, pwitte3@gatech.edu
 # Date: January 2020
 
-using InvertibleNetworks, LinearAlgebra, Test
+using InvertibleNetworks, LinearAlgebra, Test, Flux, Random
 
```

Comment:
maybe easier to toggle between cpu and gpu by defining `device = cpu` as default as assigning the necessarily variables to `|> device`.